spec/api/v1/external_users/claims/integration/advocate_claim_creation_spec.rb

```diff
@@ -613,7 +611,6 @@ def results(claim, expected)
       end
 
       include_context 'deactivate deprecation warnings'
-      include_examples 'scheme 12 advocate final claim', relative_endpoint: :advocate
```

Comment:
[ditto](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/pull/3971/commits/3bed6f8e870ba83943c8d3eaa378908981a3d790#r632620505)